pkg/asset/agent/image/agentpxefiles.go

```diff
@@ -169,3 +178,35 @@ func (a *AgentPXEFiles) copy(filepath string, src io.Reader) error {
 
 	return nil
 }
+
+func (a *AgentPXEFiles) createiPXEScript(cpuArch, pxeBaseURL, pxeAssetsFullPath string) error {
+	iPXEScriptTemplate := `
+#!ipxe
+initrd --name initrd %s/%s 
+kernel %s/%s  
+initrd=initrd 
+coreos.live.rootfs_url=%s/%sg 
+random.trust_cpu=on 
+rd.luks.options=discard 
+ignition.firstboot 
+ignition.platform.id=metal 
+console=tty1 
+console=ttyS1,115200n8 
+coreos.inst.persistent-kargs="console=tty1 console=ttyS1,115200n8" 
+boot`
+
+	iPXEScript := fmt.Sprintf(iPXEScriptTemplate, pxeBaseURL,
```

Comment:
nit: the way this is formatted, with the line breaks occurring between the strings that are on the same line in the output, is less readable than it could be.